packages/base-map/src/index.tsx

```diff
@@ -151,9 +151,29 @@ const BaseMap = ({
         <Styled.LayerSelector
           className="filter-group"
           id="filter-group"
+          onFocus={() => setFakeHover(true)}
+          onBlur={e => {
```

Comment:
There is still something going on with the blue handling. Using the story at http://localhost:5555/?path=/story/basemap--optional-layers&globals=locale:en-US, you cannot tab to toggle the second layer.

Reply:
On the story at http://localhost:5555/?path=/story/basemap--with-multiple-base-layers, the popup still cannot be dismissed.